### ISSUE TITLE
[Modal] Attempt to focus and focustrap on the first button or component not working

### ISSUE THREAD

**Issue by driosnav** [Semantic-Org/Semantic-UI#7082]
I am trying to implement a modal with focus trap using https://github.com/focus-trap/focus-trap. The modal of semantic UI in react is not accessible, so I am trying to this and put the focus in the a button that is component. the button itself is a html5 normal button. But I am not able to focus on the button. I had try the ref solution but still, the button is not focus inside the modal. Any help about this issue or it's possible to do that?

I was following this issue, but it seems to be working only for inputs, not buttons.
